init_db.py

```
@@ -0,0 +1,23 @@
+from src.infrastructure.db.engine import engine
+from src.infrastructure.db.models import SQLModel, Semester, Subject, Assignment
+
+def main():
+    print("Attempting to connect to the database...")
+    # Give the database a moment to start up
+    time.sleep(5)
+    
+    try:
+        # The engine is already created in engine.py
+        # This will attempt to connect to the database
+        with engine.connect() as connection:
```

Ruff (0.14.3)

Local variable `connection` is assigned to but never used

Remove assignment to unused variable `connection`

(F841)